aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -527,7 +572,7 @@ public void updatePassword(
             @NonNull String oldPassword,
             @NonNull String newPassword,
             @Nullable Action onSuccess,
-            @Nullable Consumer<AuthException> onError
+            @Nullable Consumer<AuthException> onException
```

Comment:
All of them in this file are using onException to avoid conflicting with AWSMobileClient's onError methods so this was making it consistent with the rest of the file.